docs/cli-reference.md

```
@@ -245,7 +247,7 @@ Every `download` subcommand accepts the same selection / safety / output flag se
 | `slide-deck [path]` | Output path | `--format [pdf\|pptx]` (default: pdf) | `download slide-deck ./slides.pdf` |
 | `infographic [path]` | Output path | (none) | `download infographic ./info.png` |
 | `report [path]` | Output path | (none) | `download report ./report.md` |
-| `mind-map [path]` | Output path | (none) | `download mind-map ./map.json` |
+| `mind-map [path]` | Output path | (none) | `download mind-map ./map.json` |<!-- both note-backed and interactive kinds export their JSON node tree -->
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)